.github/workflows/_binary-build-linux.yml

```diff
@@ -213,7 +225,7 @@ jobs:
 
       - name: Pull Docker image
         if: ${{ steps.filter.outputs.is-test-matrix-empty == 'False' }}
-        uses: pytorch/test-infra/.github/actions/pull-docker-image@main
+        uses: ./test-infra/.github/actions/pull-docker-image
```

Comment:
This works I think.  I'm curious what the reason is for checking out `test-infra` explicitly instead of using `pytorch/test-infra/.github/actions/pull-docker-image@${{ env.REPO_BRANCH }}`

Reply:
Unfortunately this will no work. Here is good discussion: https://github.com/orgs/community/discussions/45342